### ISSUE TITLE
A basic step by step for running migrations in node postgres

### ISSUE THREAD

**Issue by naazy** [dwyl/node-postgres-migrations#1]
@sohilpandya has requested that we share our knowledge on running migrations in node postgres to make it easier for other teams to get their migrations up and running

Linked to https://github.com/emfoundation/ce100-app/issues/652

**Comment by sohilpandya** [comment]
Really appreciate this @naazy ❤️ 

Here is a step by step process of running a migration as far as I understand (it's similar to what's already present in #2)

- create a **`postinstall.js`** script that will run on Heroku
- connect to your database
- run all the migration files individually on the Heroku Database.

**The caveat here is how does one know if a migration script has been run before?**

- We should have a table called `migration` in the database which stores a list of migrations that have been run before. 

So the above process is actually:
- connect to your database
- run migration on individual files only if it hasn't been run before and then save the name of the file to the migration table
- leave it if the file name already exists in the table as it's been run before.

**Comment by naazy** [comment]
That's correct @sohilpandya. However, you should be able to just copy the callMigrations.js and migrations.js and this will carry out all of the migrations functionality you mentioned above (including creating the migrations table in the db). Also you don't need a postinstall.js, literally just a 'postinstall' in your package.json: so Heroku knows to run the migration after npm installing e.g.

`postinstall: 'npm run migrations'` (like we have in the package.json in this repo)

I will make the steps in the PR clearer!
